frontend/src/views/Login.vue

```diff
@@ -65,7 +66,12 @@ export default {
       if (this.teamName === "") {
         return "Team name can't be nothing.";
       }
-      return `Sorry, the name ${this.unavailableName} is taken. Please pick another one.`;
+      return `Sorry, the name ${this.unavailableName} is taken or invalid. Please pick another one.`;
+    }
+  },
+  watch: {
```

Comment:
not sure why we need a watch here. I think it would nice if the error message (unavailableName is taken) still be on top in case the kids forget what they typed before??

Reply:
fixed